iron-collapse.html

```diff
@@ -124,6 +124,21 @@
       this._enableTransition = true;
     },
 
+    attached: function() {
+      this._contentObserver = Polymer.dom(this.$.content).observeNodes(
+        function onContentChanged(){
```

Comment:
yeah, i didn't declare the input parameter since i don't use it. No unused vars, right?